src/main/cpp/mechanisms/base/Mech1Servo.h

```diff
@@ -60,6 +60,9 @@ class Mech1Servo : public Mech
     private:
         DragonServo*                                m_servo;
 
+    protected:

```

Comment:
Shouldn't be changing the base class unless it applies to all.  What is m_test for?